src/main/scala/outwatch/advanced/package.scala

```diff
@@ -0,0 +1,17 @@
+package outwatch
+
+import rxscalajs.Observable
+
+package object advanced {
```

Comment:
I don't understand why we need an advanced package, or how these methods could be unsafe. I find these methods especially useful to write correct, bug-free code.

Do you have an example of an unsafe usage?

Reply:
It's safe as long the original `Observable` is transformed, but a different `Observable` could be returned.

For example, let's say one would like to emit two elements every time a button is clicked and writes this (incorrect) code:

```scala
Handler.create[Int] { handler =>
  val many = handler.transformSink[MouseEvent](_ => Observable.of(1,2))
  button(click --> many)
}
```

However, this doesn't work as expected because the handler's observable is closed immediately on creation and the `click` handler doesn't do anything.

Reply:
How is that unsafe? The whole observable is replaced by another one. We cannot protect the user to use the wrong operators. Am I missing something?

Reply:
Safe was not the correct term to use. What I was trying to say is that in the above example the button click doesn't do anything because the observable is completed after emitting (1, 2) , which is probably not what is intended, instead it should have been: 

```scala
Handler.create[Int] { handler =>
  val many = handler.transformSink[MouseEvent](_.concatMap(Observable.of(1,2)))
  button(click --> many)
}
```

Often the `transform...` methods will require the use of higher order observables. There's nothing wrong with that, but they are easier to use incorrectly, which is why I placed them under the `outwatch.advanced._` import. It also aligns with one of the Outwatch goal: "Removing or restricting the need for Higher Order Observables".

Reply:
I've actually changed my mind. We should place the `transform...` methods in the `HandlerOps` trait. They are no more "advanced" than `Sink.redirect`, which are available to the user.